spec/pagy_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
I'm not quite sure these tests are needed. This just seems to be testing the gem itself, which isn't what we'd want to do

Reply:
I have added them to ensure that 'Pagy' is configured the right way.
If it's not configured in the 'last_page' overflow mode it will fail.
As for the last test I agree too.
Do you need me to remove the last test or all of the tests?

Reply:
I'd personally just remove them all

Reply:
Alright, I've just removed the spec file.